dev/cuda/matmul_backward_bias.cu

```diff
@@ -288,31 +479,30 @@ int main(int argc, char **argv) {
     // matmul_backward_bias(kernel_num, NULL, NULL, d_dbias, d_dout, NULL, NULL, NULL, B, T, C, OC, block_size_debug);
     // exit(EXIT_SUCCESS);
 
-    int block_sizes[] = {32, 64, 128, 256, 512, 1024};
+    int block_sizes[] = {32, 64, 128, 256, 512, 768, 1024};
```

Comment:
I realized we should have been adding this more places btw, when I saw that some GPUs often have thresholds set on 768, e.g. number of max threads per SM on some GPUs and such.